en/advanced_config/esc_calibration.md

```diff
@@ -24,33 +38,42 @@ To calibrate the ESCs:
 1. Remove the propellers.
 
    :::warning
-   Never attempt ESC calibration with props on.
-   
+   Never attempt ESC calibration with propellers on!
+
    The motors should not spin during ESC calibration.
-   However if an ESC doesn't properly support/detect the calibration sequence then it will respond to the PWM input by running the motor at maximum speed.
+   However if the calibration starts when the ESC is powered already or the ESC doesn't properly support/detect the calibration sequence then it will respond to the PWM input by running the motor at maximum speed.
    :::
-1. Disconnect the battery and connect the flight controller via USB (only).
+1. Map the ESCs you're calibrating as Motors in the vehicle's [Actuator Configuration](../config/actuators.md). Only mapped actuators get an output and hence only ESCs mapped as motors get calibrated.
+1. Unpower the ESCs by unplugging the battery but keep the flight controller powered and connected to the ground station best by plugging it via USB (only).

```

Comment:
```suggestion
1. Unpower the ESCs by unplugging the battery.
   The flight controller should ideally still be powered via USB, and remain connected to the ground station.

```

Reply:
> flight controller should ideally still be powered

Maybe that's already covered but the flight controller must stay powered. That's easiest (= ideal?) achieved with the classic pixhawk boards because they stay powered with USB only.

For e.g. Skynode USB alone is not enough, you need to keep the power module connected and so it's more tricky. I'm not sure but wouldn't be surprised if there are other setups that work this way. Hence it's also mentioned in the troubleshooting. Hopefully, I can improve the workflow for these cases.

Reply:
I will tidy this bit.